source/common/api/win32/os_sys_calls_impl.cc

```diff
@@ -158,6 +158,10 @@ SysCallSizeResult OsSysCallsImpl::pread(os_fd_t fd, void* buffer, size_t length,
   PANIC("not implemented");
 }
 
+SysCallSizeResult OsSysCallsImpl::send(os_fd_t socket, void* buffer, size_t length, int flags) {
+  PANIC("not implemented");
```

Comment:
Why is it "not implemented"? Windows does have the `send` function. Implementing this method will avoid #ifdefs below.

Reply:
We haven't tested the impact of the change on Windows so I'm trying to keep the logic unchanged for Windows. Does the change look relatively safe for Windows? I can work on enabling it for Windows as well.